CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+Azurite, Version 2.7.1
+A lightweight server clone of Azure Storage
+
+Azure Table Storage Emulator listening on port 10002
+Azure Queue Storage Emulator listening on port 10001
+Azure Blob Storage Emulator listening on port 10000
+```
+
+### Visual Studio Code Extensions
+
+The following extensions should be installed if using Visual Studio Code for debugging:
+
+- C# for Visual Studio Code (powered by OmniSharp)
+- Azure Functions Extensions for Visual Studio Code v0.19.1 and above.
+
+### Setting up durable-js debugging
+
+- Create a Durable Functions Orchestrator for FunctionChaining pattern using [starter templates](https://docs.microsoft.com/en-us/azure/azure-functions/durable/quickstart-js-vscode)
+  Note: In this starter template, ignore the line that says: "On a Mac or Linux computer, you must set the AzureWebJobsStorage property to the connection string of an existing Azure storage account". We will be setting up the AzureWebJobsStorage property to `UseDevelopmentStorage=true`
+
+- In host.json, remove the extensionsBundle portion to enable specific version debugging. Provide a hub name (else remove the entire extensions portion to default to DurableFunctionsHub) Here's how the host.json should look like:
+
+```
+{
+  "version": "2.0",
+  "extensions": {
+    "durableTask": {
+      "hubName": "{hubName}"
+    }
+  }
+}
+```
+
+- `func extensions install` (this will install an extensions.csproj that contains the version of DurableTask as seen below)
+
+```xml <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="1.8.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.1.0" />
+  </ItemGroup>
+```
+
+### Setting up Durable Extension .NET debugging
+
+- Download the source code of DurableTask extension from [here](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask)
+- In launch.json, add requireExactSource and justMyCode fields for .NET Core Attach task.
+
+```
+ {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:pickProcess}",
+            "requireExactSource": false,
+            "justMyCode": false
+ }
+ ```
+
+### Debugging end-to-end
+
+1. Open the Azure Storage Explorer and connect to the local storage emulator or the storage account you are using.
+2. In the VSCode editor for durable-js click Debug -> Start Debugging. This will internally start `func host start` through core tools and provides the orchestrator client URL
+3. In the VSCode editor for DurableTask, click Debug -> .NET Core Attach Process and search for `func host start` process and attach to it.
+4. Add a breakpoint in both editors and continue debugging.
+
+## Testing changes locally (Windows)
+
+Follow all the steps above in Visual Studio and use Azure Storage Emulator for Windows to simulate the storage account.
```

Comment:
I would clarify that the only steps above to do in Visual Studio is the Durable Extension .NET debugging. Even then, Windows JS developers may still feel more comfortable using VS Code, so maybe reword with something like the below:

Follow all the steps above, use the Azure Storage Emulator for windows to simulate the storage account, and optionally use Visual Studio to debug the .NET Durable Extension.

Reply:
Done https://github.com/Azure/azure-functions-durable-js/pull/124/commits/7f085c2fb23d722fb07afce41cf6f653b739a2ee